src/scenes/Moves/Ppm/WeightTicket.jsx

```diff
@@ -120,7 +120,11 @@ class WeightTicket extends Component {
           )}
 
           {/* TODO: change onclick handler to go to next page in flow */}
-          <PPMPaymentRequestActionBtns onClick={() => {}} nextBtnLabel="Save & Add Another" />
+          {additionalWeightTickets === 'Yes' ? (
```

Comment:
I just meant the label itself! 
```suggestion
          <PPMPaymentRequestActionBtns nextBtnLabel={nextBtnlabel} />
```

Then above you could do
```
const nextBtnlabel = additionalWeightTickets === 'Yes' ? 'Save & Add Another' : 'Save & Continue'
```

Reply:
got it.